components/html-block/html-block.js

```diff
@@ -22,32 +25,32 @@ export const htmlBlockContentStyles = css`
 	}
 	h1 {
 		font-size: 2em;
-		line-height: 37px;
+		line-height: 0.97;
 		margin: 21.43px 0;
```

Comment:
Aside from your change... these fractional values are pretty useless imo.  I am sure they were calculated from somewhere at some point.

Reply:
Haha probably a similar thing to what I did. To keep things as close as possible to existing behaviour I just did the divisions and rounded to two decimal places. I do wonder how useful they are in the line-height case because they're all so very close to 1.